src/platform/silabs/wifi/SiWx/WifiInterface.cpp

```diff
@@ -112,11 +112,17 @@ osMessageQueueId_t sWifiEventQueue = nullptr;
 sl_net_wifi_lwip_context_t wifi_client_context;
 sl_wifi_security_t security = SL_WIFI_SECURITY_UNKNOWN;
 
+#ifdef EXP_BOARD
+#define REGION_CODE IGNORE_REGION
+#else
+#define REGION_CODE US
+#endif // EXP_BOARD
```

Comment:
This isn't necessary. It was removed with the Wiseconnect update.

Reply:
Can you add a todo with the explanation please?

Reply:
Added